Sources/PayPalMessages/Config/PayPalMessageModalConfig.swift

```diff
@@ -9,21 +9,24 @@ class ModalCloseButtonConfig: NSObject {
     var availableHeight: Int
     var color: UIColor
     var colorType: String
+    var alternativeText: String
 
     init(
         width: Int? = nil,
         height: Int? = nil,
         availableWidth: Int? = nil,
         availableHeight: Int? = nil,
         color: UIColor? = nil,
-        colorType: String? = nil
+        colorType: String? = nil,
+        alternativeText: String? = nil
     ) {
         self.width = width ?? 26
         self.height = height ?? 26
         self.availableWidth = availableWidth ?? 60
         self.availableHeight = availableHeight ?? 60
         self.color = color ?? UIColor(hexString: "#001435")
         self.colorType = colorType ?? "dark"
+        self.alternativeText = alternativeText ?? "PayPal Learn More Modal Close"
```

Comment:
default text in https://github.paypal.com/ApiSpecifications-R/credit.FinancingPresentmentSpecification/pull/25/files is "PayPal modal close" - do we want the same text or to specify 'Learn More?'